src/connectors/dynamic-widgets/__tests__/connectDynamicWidgets-test.ts

```diff
@@ -39,17 +43,12 @@ describe('connectDynamicWidgets', () => {
       `);
     });
 
-    it('correct usage', () => {
+    it('minimal usage', () => {
```

Comment:
I don't think this test makes sense anymore if we don't take into account the first implem. It's like if we were testing that adding each option doesn't throw.